libsplinter/src/biome/rest_api/config.rs

```diff
@@ -39,19 +39,27 @@ pub struct BiomeRestConfig {
 }
 
 impl BiomeRestConfig {
+    /// Returns token issuer string. Defaults to "self-issued"
```

Comment:
I feel like if it has one period, then everything has to end in periods. Maybe: "Return token issuer string (default: "self-issued")."